libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Skills/AdaptiveSkillDialog.cs

```diff
@@ -38,16 +38,6 @@ public AdaptiveSkillDialog([CallerFilePath] string callerPath = "", [CallerLineN
         [JsonProperty("disabled")]
         public BoolExpression Disabled { get; set; }
 
-        /// <summary>
-        /// Gets or sets a value indicating whether to have the new dialog should process the activity.
-        /// </summary>
-        /// <value>
-        /// The default for this will be true, which means the new dialog should not look the activity. You can set this to false to dispatch the activity to the new dialog.
-        /// </value>
-        [DefaultValue(true)]
-        [JsonProperty("activityProcessed")]
-        public BoolExpression ActivityProcessed { get; set; } = true;
-
         /// <summary>
```

Comment:
You shouldn't remove this.  If you do this you have no way of controlling whether the skill should be doing recognition on the activity or not.

Reply:
@luhan2017 requested we remove this and @vishwacsena also agreed (see issue linked to the PR), we don't seem to be understand how is being used by composer or skilldialog.

Reply:
I don't see how this activityProcessed is passed to and consumed by the skill bot. Do we pass the turn memory of the caller bot to the skill bot? 
In the SkillDialog, the activity is not from the last turn, it just comes from the property of the SkillDialog, so I feel that the meaning of the activityProcessed in BeginDialog is not same as the meaning of whether we need to go through intent recognition in skill, should we make another setting specially for this?

Reply:
I added code in AdaptiveSkillDialog to use the ActivityProcessed property and updated the code documentation to describe its behavior.

Reply:
Hi @luhan2017, ping me over Teams if you want to chat about this one a bit more.